src/App.js

```diff
@@ -1,6 +1,6 @@
 import "antd/dist/antd.css";
 import "./App.css";
```

Comment:
CSS imports should occur in the end of all imports and add a space after imports and before starting the function